backend/controllers/game-controllers.js

```diff
@@ -193,20 +199,23 @@ function removePlayerFromGame(roomId, playerId) {
 }
 
 const startRound = (io, roomId) => {
-  const game = games[roomId];
+  let game = games[roomId];
   const currentQuestion = game.questions[game.currentQuestionIndex];
   const sentQuestion = JSON.parse(JSON.stringify(currentQuestion));
   delete sentQuestion["correctAnswer"];
   io.to(roomId).emit("nextQuestion", sentQuestion);
+  game["gameStarted"] = true;
+  console.log(game["gameStarted"]);
 
   const roundDuration = game.gameRules.snippetLength;
   let timeLeft = roundDuration;
   const roundTimer = setInterval(() => {
     io.to(roomId).emit("timerTick", { timeLeft });
     timeLeft--;
 
-    if (timeLeft < 0) {
+    if (timeLeft < 0 || answered === Object.values(game.players).length) {
```

Comment:
potential issue if we do this. If a player runs out of time, and isn't able to answer the question, they can never answer the question once times runs out so the next round will never start. 

I think we should just make it so the round ends based on a timer, rather than if all players answered, so the game isn't locked by one person